packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -97,6 +113,15 @@ class GoogleSignInAccount implements GoogleIdentity {
     return GoogleSignInAuthentication._(response);
   }
 
+  Future<GoogleHttpClient> get httpClient async {
```

Comment:
This whole file is lacking proper comments. Could you please start the snowball by commenting yours? :)

Reply:
sure!